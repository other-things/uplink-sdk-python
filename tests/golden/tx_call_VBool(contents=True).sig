r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 29703960519957442228530340157391955047145525081017981250665037713661720497697
