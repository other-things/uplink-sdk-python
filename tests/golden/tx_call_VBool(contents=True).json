{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
                "args": [
                    {
                        "contents": true,
                        "tag": "VBool"
                    }
                ],
                "method": "get"
            },
            "tag": "Call"
        },
        "tag": "TxContract"
    },
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE0yOTcwMzk2MDUxOTk1NzQ0MjIyODUzMDM0MDE1NzM5MTk1NTA0NzE0NTUyNTA4MTAxNzk4MTI1MDY2NTAzNzcxMzY2MTcyMDQ5NzY5Nw=="
}